### PR TITLE
[CDAP-7462] Integrating PlusButton across webapps

### PR DESCRIPTION
**Pull request by ajainarayanan**
### Things added
- Splits webpack config for login & cdap webapps 
- Makes PlusButton as a standalone library to be used across webapps
- Adds a separate webpack config for plusbutton library
- Updates fonts for hydrator & tracker (to use new fonts from cdap)
- Exports `PlusButton` and `Store` (namespace store) as part of the library.
  So when the plusbutton library is loaded into the browser it will add `PlusButton` object to the `window` which is comprised of `default` and `Store`. `default` being the component.
### TODO
- Make plusbutton library size smaller in size.
- Unify fonts and styles to be re-used across webapps. Right now its a copy paste and its bad and ugly and redundant.
- Improve the gulp process. Should we even use gulp to run webpack for building the plus button? Need to see if we can move webpack stuff out of gulp. What happens if I want to have a watch mode in webpack and I am doing a `gulp watch`? Right now its a hack and we should consider cleaning that out.

### CODE REVIEW

File: cdap-ui/app/plusbutton/plusbutton.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import T from 'i18n-react';
+T.setTexts(require('../cdap/text/text-en.yaml'));
+// import PlusButton from '../cdap/components/PlusButton';
```
Comment (elfenheart) [inline cdap-ui/app/plusbutton/plusbutton.js]: remove?

File: cdap-ui/app/styles/themes/cdap/theme.less [hunk 2]
```diff
@@ -390,185 +390,188 @@ body.theme-cdap {
...
       }
-    }
-    // Small modals (ie, cask-confirmable, cask-promptable)
-    .modal-sm {
-      // max-width: 630px;
-      text-align: center;
-      .modal-footer {
+      // Small modals (ie, cask-confirmable, cask-promptable)
+      .modal-sm {
+        // max-width: 630px;
```
Comment (PGuimarais) [inline cdap-ui/app/styles/themes/cdap/theme.less]: remove?

**Comment by elfenheart** [comment]
screen shot 2016-10-06 at 2 37 51 pm

Font in Tracker seemed different for Upload Artifact Wizard?

**Comment by ajainarayanan** [comment]
@elfenheart For some reason I am not seeing this. Is there a sequence of steps to reproduce this? (Or just open tracker and open the wizard?)

File: cdap-ui/app/plusbutton/plusbutton.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/plusbutton/plusbutton.js]: Removed

File: cdap-ui/app/styles/themes/cdap/theme.less (hunk 2 above)
Comment (ajainarayanan) [reply cdap-ui/app/styles/themes/cdap/theme.less]: Done. Removed.

**Comment by elfenheart** [comment]
@ajainarayanan it was just opening the wizard. Let me do a clean UI build and try again

**Comment by elfenheart** [comment]
@ajainarayanan Still seemed to have the problem with Tracker. It seems like it lost specificity for styling for <span>. This happens in Tracker Home page btw.

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**